src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -285,18 +288,10 @@ void AtlasEngine::_handleSwapChainUpdate()
 {
     if (_p.swapChain.targetGeneration != _p.s->target.generation())
     {
-        if (_p.swapChain.swapChain)
-        {
-            _b->ReleaseResources();
-            _p.deviceContext->ClearState();
-            _p.deviceContext->Flush();
-        }
         _createSwapChain();

```

Comment:
This failed to release the swap chain itself before clearing and flushing the device. That's just not right, I think. I'm not sure what effect this could've had (because it never led to any issues for me), but it's definitely not allowed according to D3D11.

`_createSwapChain` will now call `_destroySwapChain`.